src/core/ext/transport/chttp2/transport/chttp2_transport.c

```diff
@@ -2398,6 +2418,36 @@ void grpc_chttp2_config_default_keepalive_args(grpc_channel_args *args,
                 &args->args[i],
                 (grpc_integer_options){g_default_keepalive_permit_without_calls,
                                        0, 1});
+      } else if (0 ==
+                 strcmp(args->args[i].key, GRPC_ARG_HTTP2_MAX_PING_STRIKES)) {
+        g_default_max_ping_strikes = grpc_channel_arg_get_integer(
```

Comment:
What does initialization stage mean?
- before grpc_init?
- after grpc_init?
- before transport construction?

If it's an internal interface, who is supposed to call it? (no code can, because it's internal)

What is the justification for introducing yet another configuration mechanism?